### PR TITLE
Add new AcadeMeData app

### PR DESCRIPTION
**Pull request by DanWaldman**
- Includes all files generated by startapp command.
- Does not include models - will come in later PR's.
- Added .idea file to git ignore.
- Added app to installed apps in settings.py.

### CODE REVIEW

**Comment by Yarboa** [comment]
@DanWaldman please review the git challange again
As a reviewer i can not track the history, since you are creating new PR each time 
https://github.com/redhat-beyond/AcadeMe/pull/28, which is closes

File: AcadeMeData/.idea/AcadeMeData.iml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Yarboa) [inline AcadeMeData/.idea/AcadeMeData.iml:1]: Please add the .idea to .ignore file

File: AcadeMeData/.idea/inspectionProfiles/profiles_settings.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (Yarboa) [inline AcadeMeData/.idea/inspectionProfiles/profiles_settings.xml:1]: same here for the ignorefile

**Review by Yarboa — CHANGES_REQUESTED** [review]

**Comment by Omeramsc** [comment]
As I mentioned before - please add an issue about this feature, and make sure both the issue, commit message and PR have a well-written description which someone who don't know the project can understand. 

- also, make the titles of the PRs shorter and clearer. an example: `Add new AcadeMeData app module`

**Comment by DanWaldman** [comment]
An Issue has been opened about this feature.
A new commit adding .idea to ignore file has been uploaded. 

Can the PR be merged now?

**Comment by Yarboa** [comment]
> An Issue has been opened about this feature. A new commit adding .idea to ignore file has been uploaded.

Thanks
But still you should apply git rm .idea dir which the files already exist in PR
> 
> Can the PR be merged now?
No see above,
An no comment in this PR

**Comment by DanWaldman** [comment]
FINALLY - I've rearranged my commits for them to have a good order and no repetitions.
I added .idea to git ignore and made those filed go away from the depository.
I added all files that were created by startapp command. 
Hopefully models will be coming in next PR's.

**Comment by DanWaldman** [comment]
#34

File: AcadeMeData/apps.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class AcademedataConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
```
Comment (Yarboa) [inline AcadeMeData/apps.py:5]: Why do you need this one?

Comment (DanWaldman) [reply AcadeMeData/apps.py:5]: I didn't add this, I assumed it automatically was created by startapp command so I didn't remove it.

Comment (Yarboa) [reply AcadeMeData/apps.py:5]: > I didn't add this, I assumed it automatically was created by startapp command so I didn't remove it.

Please remove it

Comment (Yarboa) [reply AcadeMeData/apps.py:5]: > I didn't add this, I assumed it automatically was created by startapp command so I didn't remove it.

Please remove it

Comment (DanWaldman) [reply AcadeMeData/apps.py]: Done

**Review by Yarboa — APPROVED** [review]

**Pull request merged**